### PR TITLE
feat(core): add position to mapped input/output streams of a workflow

### PR DESCRIPTION
**Pull request by Panaetius**
adds position to inputs/outputs that were mapped to stdin/stdout/stderr

### CODE REVIEW

File: renku/core/management/workflow/plan_factory.py [hunk 1]
```diff
@@ -395,6 +397,12 @@ def add_command_output(
 
         mapped_stream = self.get_stream_mapping_for_value(default_value)
 
+        if mapped_stream and not position:
```
Comment (mohammad-sdsc) [inline renku/core/management/workflow/plan_factory.py:400]: Does it matter if we check `not position` instead of `position is None`?

**Review by mohammad-sdsc — DISMISSED** [review]
LGTM, made a single comment. Feel free to ignore if not relevant.

File: renku/core/management/workflow/plan_factory.py (hunk 1 above)
Comment (Panaetius) [reply renku/core/management/workflow/plan_factory.py:400]: it shouldn't, position always starts at 1. but `is None` is cleaner, I'll change it

**Review by mohammad-sdsc — APPROVED** [review]
Thank you!

**Pull request merged**